src/common/interop/AssemblyInfo.cpp

```diff
@@ -0,0 +1,22 @@
+#include "pch.h"
+
+using namespace System;
+using namespace System::Reflection;
+using namespace System::Runtime::CompilerServices;
+using namespace System::Runtime::InteropServices;
+using namespace System::Security::Permissions;
+
+[assembly:AssemblyTitleAttribute(L"interop")];
```

Comment:
Does this project generate a dll that would have to be included in the MSI? If so, I think we should fill in the assembly info similar to PowerToys. For all the C# projects in PowerToys master this file is autogenerated from code written in the csproj file. You can check this file for reference, we might be able to do something similar here like [FancyZones](https://github.com/microsoft/PowerToys/blob/dcee505f6b8b6980aa1f8a9adcaecdab4071d747/src/modules/fancyzones/editor/FancyZonesEditor/FancyZonesEditor.csproj#L14-L35). That will ensure stuff like version number are in sync with the rest of the power toys.

Reply:
I updated the project file to do AssemblyInfo generation.